docs/5-design-patterns/hometask/client.ts

```diff
@@ -0,0 +1,36 @@
+const mockClient = {
+    weight: 1,
+    fromAddress: 'address 1',
+    fromZipCode: '1',
+    toAddress: 'address 2',
+    toZipCode: '1',
+    weight2: 17,
+    typesMarks: ['Fragile', 'Do Not Leave']
+};
+
+export class Client {
+    id?: number;
```

Comment:
All properties of class usually are places as constructor arguments. So to create Client instance you need to call new Client(3345, 54, blabla....) and specify all arguments. In you implementation how exactly you will create different instances?